Source/Orts.Simulation/Simulation/RollingStocks/MSTSSteamLocomotive.cs

```diff
@@ -4383,7 +4383,10 @@ private void UpdateCylinders(float elapsedClockSeconds, float throttle, float cu
                     CylinderAdmissionSteamWeightLbs = 0.0f;
                 }
                 RawCylinderSteamWeightLbs = CylinderReleaseSteamWeightLbs - CylinderAdmissionSteamWeightLbs;
-                RawCalculatedCylinderSteamUsageLBpS = NumCylinders * DrvWheelRevRpS * CylStrokesPerCycle * (RawCylinderSteamWeightLbs);
+
+                // Calculate steam usage based upon how many piston strokes happen ffor each revolution of the wheels. 
+                // Geared locomotives will have to take into account gearing ratio. 
+                RawCalculatedCylinderSteamUsageLBpS = NumCylinders * DrvWheelRevRpS * MotiveForceGearRatio * CylStrokesPerCycle * (RawCylinderSteamWeightLbs);
```

Comment:
```suggestion
                RawCalculatedCylinderSteamUsageLBpS = NumCylinders * DrvWheelRevRpS * MotiveForceGearRatio * CylStrokesPerCycle * RawCylinderSteamWeightLbs;
```
Suuuuuper minor thing: Let's drop the extraneous parantheses while we're here. 🙂

Reply:
Done